src/Tribe/Aggregator/Record/Queue_Realtime.php

```diff
@@ -35,7 +35,9 @@ public function __construct(
 	) {
 		tribe_notice( 'aggregator-update-msg', array( $this, 'render_update_message' ), 'type=warning&dismiss=0' );
 
+		add_filter( 'heartbeat_received', [ $this, 'receive_heartbeat' ], 10, 2 );
 		add_action( 'wp_ajax_tribe_aggregator_realtime_update', array( $this, 'ajax' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 7d234b2cc427e8062872a5976d31178c</sub>